runtime/ftplugin/roc.vim

```diff
@@ -0,0 +1,20 @@
+" Roc filetype plugin file
+" Language: Roc
+" Maintainer: nat-418 <93013864+nat-418@users.noreply.github.com>
+" Latest Revision: 2024-04-5
+
+if exists("b:did_ftplugin")
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo&vim
```

Comment:
I think there is nothing in your plugin requiring the 
```
let s:cpo_save = &cpo
set cpo&vim
```
dance and you can remove it.